website/docs/docs/guides/migration-guide/migrating-from-spark-to-databricks.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Migrating from dbt-spark to dbt-databricks"
+id: "migrating-from-spark-to-databricks"
+---
+
+
+## Pre-requisites
+
+In order to migrate to dbt-databricks, you must first be using `dbt-spark>1.0.0` as `dbt-databricks` is only supported from `>1.0`.
+
+## Why change to dbt-databricks?
+
+Current users of dbt with Databricks are about to have some new experiences with respect to authentication as well as features available via the delta format.
+
+### Authentication Simplification
+
+The authentication information has now simplified greatly. Previously you would have to provide a `cluster` or `endpoint` id which was hard to parse out of the http_path provided in the Databricks UI. Now it is simple enough  changes are authentication method is now the same regardless if you are using a Cluster or a SQL endpoint
```

Comment:
super nit: an extra space between `simple enough` and `changes are`.

Reply:
totally valid!